app/src/pages/trace/TracePage.tsx

```diff
@@ -445,11 +446,21 @@ function SpanInfo({ span }: { span: Span }) {
   const { json: attributesObject, parseError } =
     useSafelyParsedJSON(attributes);
 
+  let statusDescription: ReactNode;
+  if (span.statusMessage) {
+    statusDescription = (
+      <Alert variant="warning" title="Status Description">
+        {span.statusMessage}
+      </Alert>
+    );
+  }
```

Comment:
thought, you can wrap this in useMemo to avoid re-render. Not necessary but good practice

Reply:
updated